spec/lib/engine/games/game_spec.rb

```diff
@@ -147,16 +147,23 @@ module Engine
       results.each do |game_id, result|
         context game_id do
           it 'matches result exactly' do
-            game_path = game.title.gsub(/(.)([A-Z])/, '\1_\2').downcase
-            data = JSON.parse(File.read("spec/fixtures/#{game_path}/#{game_id}.json"))
-            players = data['players'].map { |p| p['name'] }
-            expect(game.new(players, id: game_id, actions: data['actions']).result).to eq(result)
-            rungame = game.new(players, id: game_id, actions: data['actions'], strict: true)
-            expect(rungame.result).to eq(result)
-            expect(rungame.finished).to eq(true)
+            loaded_game = load_game_fixture(game.title, game_id)
```

Comment:
i don't think we need to do this whole thing, would be cleaner for this one of situation to jmake a pure unit test

Reply:
That's fair, I was just thinking down the road of loading other 'erroneous' game states as checks. I can remove this if desired.

Reply:
If it helps your thoughts on it some, I added some bits to make it lazy-evaluating, so we're not actually evaluating a JSON game state unless explicitly requested.